tests/Aspire.Hosting.Azure.Tests/AzureAIFoundryExtensionsTests.cs

```diff
@@ -147,4 +147,19 @@ public async Task AddAzureAIFoundry_GeneratesValidBicep()
         await Verify(manifest.BicepText, extension: "bicep")
             .AppendContentAsFile(rolesManifest.BicepText, "bicep");
     }
+
+    [Fact]
+    public void AddAsExistingResource_ShouldBeIdempotent_ForAzureAIFoundryResource()
```

Comment:
Could we cut down on some duplication in tests by having a Theory that created each kind of Azure resource?

Reply:
I think we want these tests to eventually be in their own projects (well not in this repo, so keeping tests sepearate is better).